managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedger.java

```diff
@@ -138,6 +138,18 @@
      */
     public ManagedCursor openCursor(String name) throws InterruptedException, ManagedLedgerException;
 
+    /**
+     * Open a ManagedCursor in this ManagedLedger.
+     * <p>
+     * If the cursors doesn't exist, a new one will be created and its position will be at the end of the ManagedLedger.
```

Comment:
nit: its "default" position will be at the end of .....

Reply:
that's better, I'll add it

Reply:
I've already added the description in the newest commit. THX for your advice.